sys/include/net/gnrc/tcp/tcb.h

```diff
@@ -40,47 +39,52 @@ extern "C" {
 #endif
 
 /**
+ * @brief Size of the tcbs internal message queue
+ */
+#define GNRC_TCP_TCB_MSG_QUEUE_SIZE (8U)
+
+/**
  * @brief transmission control block of gnrc_tcp
  */
 typedef struct _transmission_control_block {
-    uint8_t address_family;                    /**< Address Family of local_addr and peer_addr */
+    uint8_t address_family;   /**< Address Family of local_addr and peer_addr */
 #ifdef MODULE_GNRC_IPV6
```

Comment:
I still don't understand the reasoning for this `#ifdef`, addresses are mandatory only sizes may differ. I suggest to remove this `#ifdef` and make the size large enough to fit any valid and possible address, i.e. IPv6 addresses. You could use the macro `UNIVERSAL_ADDRESS_SIZE` from `universal_address.h`.